src/components/Section/Citizenship/Status/Status.jsx

```diff
@@ -855,7 +855,7 @@ export class Status extends Subsection {
                 <Radio
                   name="document-type-visa"
                   label={i18n.t('citizenship.status.label.documenttype.visa')}
-                  value="U.S. Visa"
+                  value="U.S. Visa (red foil number)"
```

Comment:
@ryanhofdotgov @gidjin I'm not exactly sure how to word this, but shouldn't the value still be `U.S. Visa`?

Basically the issue is that @joshmfranklin realized was that the label for `U.S. Visa` was wrong. See image below:
![screen shot 2019-03-05 at 12 21 10 pm](https://user-images.githubusercontent.com/8367504/53834968-2b191900-3f41-11e9-9edc-7462f0144a77.png)

The label on the checkbox can be different from the value we send to the backend. Should the value we send change to `U.S. Visa (red foil number)`? Or should it remain as `U.S. Visa`?

Reply:
The value should remain `U.S. Visa` – the legal-approved text shouldn't affect the underlying way we encode the choices.

A larger annoyance is that the enums/checkbox values across the form are often needlessly different from what the SF-86 XML uses, so we have a layer of mapping from eApp to eQIP enum/constant values. e.g., "Top Secret" vs "TopSecret". At some point it would be nice to unify those.